src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -36,6 +37,14 @@ type Props = {
     onVersionRestore: VersionActionCallback,
     parentName: string,
     versionId?: string,
+    versionUpsellExperience?: string,
+};
+
+// Epoxy Experiment Bucks
```

Comment:
Do we need to reference `Epoxy` or `Bucks`?

Reply:
nope! Forgot to get rid of this comment!